src/app/components/JumpTo/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '../react-testing-library-with-providers';
+import JumpTo, { JumpToProps } from './index';
+import pidginArticleFixtureWithJumpToBlock from './fixtureData';
+
+describe('JumpTo Component', () => {
```

Comment:
Should maybe add a test to ensure the click/view tracking gets fired as that will be a requirement for the experiment.